src/webui/api/appcontroller.cpp

```diff
@@ -250,6 +254,66 @@ void AppController::preferencesAction()
     data["rss_processing_enabled"] = RSS::Session::instance()->isProcessingEnabled();
     data["rss_auto_downloading_enabled"] = RSS::AutoDownloader::instance()->isProcessingEnabled();
 
+    // Advanced settings
```

Comment:
Just throwing this idea out for discussion. Does anyone else think we could benefit from putting these settings behind a separate api? I think the segmentation makes sense from an api perspective, and the preferences api just continues to grow and grow. It's a bit more logic for our front end, but its justified by the api benefits.

Reply:
In my opinion, it's more reasonable to put each option into a separate api. 

But on the other hand, it may degrade performance. When we open the options dialog, it has to send a lot of HTTP requests to get all of the options, instead of one.